index.js

```diff
@@ -58,7 +59,7 @@ class Provider {
 
     const { argTypes, defaults } = this.config
     Object.keys(argTypes).map(k => {
-      if (!argTypes || !argTypes[k]) {
+      if (!parsed || !parsed[k]) {
```

Comment:
Well here you want to check if all the `args` in the `config` are parsed, right? So what should be done is:
 - parse the `argString` to an object
 - merge the default args from `config` with the parsed `argString`
 - loop through the needed arguments
    - check if a needed argument exists
       - if it does 👍 
       - if it does not 👎